pkg/odo/cli/component/create.go

```diff
@@ -294,13 +296,46 @@ A full list of component types that can be deployed is available using: 'odo cat
 	},
 }
 
+func ensureAndLogProperMemoryUsage() {
```

Comment:
:+1: For this function

Reply:
This is being updated :P, meaning I will make one generic function to work in both cases since `code climate` is complaining :)